README.md

```diff
@@ -31,7 +31,10 @@ $ cat package.json
 }
 
 $ npx sort-package-json
-package.json is sorted!
```

Comment:
Let change the lang to console, so we don't need add `#` before output.

Reply:
So create a new code block with lang of console? is this a change we want to make though the entire readme?